cmd/ovsdb-mon/shell.go

```diff
@@ -338,3 +373,31 @@ func colordiff(a, b interface{}) string {
 	}
 	return strings.TrimRight(buf.String(), "\n")
 }
+
+// filterAPI returns the conditional API that filters based on the provided filter expression
+// Expression is [FIELD]=[VALUE]
+func (s *OvsdbShell) filterAPI(tableName string, expr string) (client.ConditionalAPI, error) {
+	condModel, err := s.dbModel.NewModel(tableName)
+	if err != nil {
+		return nil, err
+	}
+	parts := strings.Split(expr, "=")
+	if len(parts) != 2 {
+		return nil, fmt.Errorf("Invalid filter expression: %s", expr)
+	}
+
+	field := parts[0]
+	value := parts[1]
+
+	fieldVal := reflect.ValueOf(condModel).Elem().FieldByName(field)
```

Comment:
heh... can you please make this case insensitive? I'm getting spoiled. ;)

Reply:
LOL of course!